script.js

```diff
@@ -23,3 +23,239 @@ links.forEach((link) => {
     document.body.style.overflow = 'visible';
   });
 });
+/** ***************************** *
+Code for Dinamically displaying the cards.
+********************************** */
+
+const cards = [
```

Comment:
- [ ] Great job so far. Your popup works well. please kindly make your card bigger and follow the figma design.
your version.
![Screenshot (256)_LI](https://user-images.githubusercontent.com/65298048/179919863-48cc63d3-6cb2-4e2b-92a2-6333f4360ca7.jpg)
figma design
![Screenshot (257)_LI](https://user-images.githubusercontent.com/65298048/179920163-572b621b-66ed-4da8-953c-72725e977a70.jpg)